### ISSUE TITLE
attempt to index global 'socket' (a nil value)

### ISSUE THREAD

**Issue by Asalle** [stackmachine/lovetest#2]
Hello there,

first of all, thanks for the awesome framework you made, I love it.

But I have a little problem with importing socket, I did everything you said on the readme, but after trying to run `love . --test` I encountered the following error: 
```
Error: test/lunatest.lua:64: attempt to index global 'socket' (a nil value)
stack traceback:
	[string "boot.lua"]:637: in function '__index'
	test/lunatest.lua:64: in main chunk
	[C]: in function 'require'
	test/lovetest.lua:26: in function 'run'
	main.lua:27: in function 'load'
	[string "boot.lua"]:488: in function <[string "boot.lua"]:487>
	[C]: in function 'xpcall'
	[string "boot.lua"]:650: in function <[string "boot.lua"]:639>
	[C]: in function 'xpcall'

```
Adding explicit import to `lunatest.lua` solved the problem.
```
local socket = require("socket")
```

**Comment by kyleconroy** [comment]
I attempted a fix here: https://github.com/stackmachine/lovetest/commit/895e5fcf180ca42fa4b31af76d0b9227fa91e7e4

Can you confirm that it works?

Also, as you can see, this repository hasn't been updated in ages. Are you interested in being a maintainer?

**Comment by Asalle** [comment]
nah, I tried to import it with pcall, it didn't work and your commit doesn't work either.
about the maintaining: maybe, if you have some time for answering my questions from time to time. I'm pretty new to lua and love2d, I started like 3 days ago.
